doc/integrator/headers.rst

```diff
@@ -30,6 +36,58 @@ Where ``<path>`` can be: ``main``, ``admin``, ``apihelp`` or ``c2c``,
 ``connect_src`` or ``worker_src``,
 ``[_extra]`` is a suffix to be able to extend a directive instance of completely overriding it.
 
+Access-Control-Allow-Origin
+---------------------------
+
+To add domain that can access to the authenticated routes you should have something like that
```

Comment:
Suggestion:
To allow additional domains to access your geoportal's authenticated areas, you can add a configuration as follows:

Reply:
I'm not about it, does "*" only works for public routes ? and authenticated routes needs precise domain to be present ?
If it is the case we might say it here.